content/en/developers/service_checks/dogstatsd_service_checks_submission.md

```diff
@@ -22,18 +22,18 @@ While StatsD accepts only metrics, DogStatsD accepts all three of the major Data
 After [installing DogStatsD][1], you can send service checks to Datadog with the following function:
 
 ```
-service_check(Name, Status, Tags, Hostname, Message)
+service_check("<SERVICE_CHECK_NAME>", <STATUS>, "<TAGS>", "<HOSTNAME>", "<MESSAGE>")
```

Comment:
nit: I would not put quotes here around the template vars, since quotes denote that they're literal strings, i.e. instead I would write:
 
```
service_check(<SERVICE_CHECK_NAME>, <STATUS>, <TAGS>, <HOSTNAME>, <MESSAGE>)
```